.github/workflows/check.yaml

```
@@ -25,13 +25,12 @@ jobs:
     secrets:
       REPO_GITHUB_TOKEN: ${{ secrets.REPO_GITHUB_TOKEN }}
   coverage:
-    if: github.event_name == 'pull_request'
-    name: Coverage 📔 
+    name: Coverage 📔
```

```suggestion
    if: github.event_name != 'push'
    name: Coverage 📔
```
What with `if` condition?

---

As discussed, let's enable coverage check also for pushes to `main`.